src/custom/utils/tip.test.ts

```diff
@@ -0,0 +1,268 @@
+/* eslint-disable @typescript-eslint/no-non-null-assertion */
+import { FeeInformation } from '@src/custom/utils/operator'
+import {
+  ChainId,
+  Currency,
+  CurrencyAmount,
+  JSBI,
+  Pair,
+  Route,
+  Token,
+  TokenAmount,
+  Trade,
+  TradeType
+} from '@uniswap/sdk'
+import { parseUnits } from 'ethers/lib/utils'
+import { computeTradePriceBreakdown } from '../../utils/prices'
+
+// const fiveMinutesFromNow = () => new Date(Date.now() + 300000).toISOString()
+// try to parse a user entered amount for a given token
+
+// had to rebuild here as import was failing...
```

Comment:
Worth looking into this

Reply:
it is due to `craco` - doesn't allow aliasing in jest. Meaning we either:
1. eject craco (no)
2. rewrite the fns we need in tests...
3. ???